src/osd/OSD.cc

```diff
@@ -211,9 +211,9 @@ OSDService::OSDService(OSD *osd) :
   op_wq(osd->op_shardedwq),
   peering_wq(osd->peering_wq),
   recovery_wq(osd->recovery_wq),
-  recovery_gen_wq("recovery_gen_wq", cct->_conf->osd_recovery_thread_timeout,
+  recovery_gen_wq("recovery_gen_wq", "wq_recov_gen", cct->_conf->osd_recovery_thread_timeout,
 		  &osd->recovery_tp),
-  op_gen_wq("op_gen_wq", cct->_conf->osd_recovery_thread_timeout, &osd->osd_tp),
+  op_gen_wq("op_gen_wq", "wq_gen_op", cct->_conf->osd_recovery_thread_timeout, &osd->osd_tp),
```

Comment:
Sometimes It looks funny, but following I wanted to keep thread naming convention.